### DISCUSSION TITLE
[Core Protocol] chaindata size is abnormally larger than other nodes.

### DISCUSSION THREAD

**Discussion by pengin7384** [ethereum-optimism/developers#334] [Core Protocol]
### Did you check the documentation?

- [X] Yes

### Did you read the specs?

- [ ] Yes
- [ ] No

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I'm running 4 archive op-geth nodes with the same configuration.
However, the chaindata size of one node becomes larger than that of other nodes.
I don't know why one node is getting larger.

One strange thing is that the size of chaindata is 8.9TB, but when I check it through `debug.chaindbProperty("");`, it is 4.7TB. Could this be relevant?

<img width="156" alt="image" src="https://github.com/ethereum-optimism/developers/assets/32831939/a32efdd5-d903-4457-a9ae-6f2244f65572">
<img width="701" alt="image" src="https://github.com/ethereum-optimism/developers/assets/32831939/80b490bc-e735-4c26-a218-8ddb0a8cf592">

### op-geth command
```
op-geth \
  --op-network=op-mainnet \
  --ws \
  --ws.port=8546 \
  --ws.addr=0.0.0.0 \
  --ws.origins="*" \
  --http \
  --http.port=8545 \
  --http.addr=0.0.0.0 \
  --http.vhosts="*" \
  --http.corsdomain="*" \
  --http.api="eth,debug,net,web3" \
  --datadir=/home/ubuntu/data/op-geth-data \
  --verbosity=3 \
  --rollup.disabletxpoolgossip=true \
  --rollup.sequencerhttp=https://mainnet-sequencer.optimism.io/ \
  --nodiscover \
  --syncmode=full \
  --gcmode=archive \
  --maxpeers=0 \
  --rollup.historicalrpc=http://localhost:8547
```

**Comment by sbvegan** [comment]
Hey @pengin7384, sorry for the delayed response. This is very odd, I'm not sure why the database would be 2x the size. The configuration you provided seems valid. Can you compare the datadir directories from your other nodes and see if there is any discrepancies. Perhaps providing a high level `tree` output, we might be able to isolate whats going on here. Also does each node have a unique path for their `--datadir`?
